packages/gatsby-theme-docs/src/plugins/remark-mdx-mermaid.js

```diff
@@ -0,0 +1,18 @@
+const visit = require('unist-util-visit');
+
+/**
+ * Given the MDXAST ast, look for all fenced codeblocks that have a language of
```

Comment:
This design choice is here for two reasons:
 - people will get used to being able to use fenced code blocks for mermaid diagrams in GitHub
 - you get syntax highlighting in the editor if it's a fenced code block but not if the syntax would be the body content of an <Mermaid> tag in MDX